brave.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/sh
+export TMPDIR="$XDG_RUNTIME_DIR/app/$FLATPAK_ID"
+exec /app/extra/brave-browser --test-type --no-sandbox "$@"
```

Comment:
What's the purpose of the `--test-type` parameter here?

According to [this list](https://peter.sh/experiments/chromium-command-line-switches/#test-type), it looks like it's just for running the Chromium tests:

> Type of the current test harness ("browser" or "ui").

Reply:
It hides the `--no-sandbox` warning.  If you don't use `--test-type` you will be given the following message every time you open up Brave browser/Chromium : 
>  You are using an unsupported command-line flag: --no-sandbox. Stability and security will suffer. 

I don't know of a better way to get rid of the message.

Reply:
Try `brave-browser --no-sandbox` and `brave-browser --test-type --no-sandbox` and you'll understand what I mean.  

It would be better if there was a better/cleaner way of doing this..

Reply:
https://source.chromium.org/chromium/chromium/src/+/master:chrome/browser/ui/startup/startup_browser_creator_impl.cc;l=743

This is the Chromium code that disables the warning. `--test-type` is the only choice I have.   

If I use `--enable-automation` then a warning would show. `--test-type` is the only flag that disables the warning and doesn't notify the user that it's set.

Reply:
I see. It might be best to avoid that option and to just live with the warning. `--test-type` is even less of a supported configuration than just `--no-sandbox`, and I don't know what the consequences of using it might be. There's also no guarantee that it will keep working that way since it's only meant for running tests.

Reply:
I think it's best to just revert to using `zypak` even though it's broken. The developer explained that `zypak` was actually sandboxing for chromium and that it wasn't just providing a fake useless sandbox.  

Basically with `zypak` Chrome's sandboxed processes have no access to the internet, files, etc. which is the goal of the regular Chromium sandbox.  It works by using `flatpak-spawn`   https://docs.flatpak.org/en/latest/flatpak-command-reference.html#flatpak-spawn

Reply:
I'm not sure about whether I should use `--no-sandbox` or use `zypak`.  

Security in something like a web browser is pretty important but having it work is just as important